indigo_api/models/works.py

```diff
@@ -262,6 +266,18 @@ def uncommenced_provisions(self):
 
         return [p for p in provisions if p.id not in commenced]
 
+    def commencements_relevant_to_current_expression(self, date):
```

Comment:
If this is on a work, the word `current` is confusing, since the work doesn't have a current expression. Perhaps just `commencements_relevant_at_date`

Reply:
good point, thanks! refactored throughout